src/test/crimson/test_messenger.cc

```diff
@@ -198,19 +372,19 @@ int main(int argc, char** argv)
      "number of pingpong rounds")
     ("keepalive-ratio", bpo::value<double>()->default_value(0.1),
      "ratio of keepalive in ping messages");
-  return app.run(argc, argv, [&] {
-    auto&& config = app.configuration();
-    verbose = config["verbose"].as<bool>();
-    auto rounds = config["rounds"].as<unsigned>();
-    auto keepalive_ratio = config["keepalive-ratio"].as<double>();
-    return test_echo(rounds, keepalive_ratio)
-    .then([] {
-      return test_concurrent_dispatch();
-    }).then([] {
-      std::cout << "All tests succeeded" << std::endl;
-    }).handle_exception([] (auto eptr) {
-      std::cout << "Test failure" << std::endl;
-      return seastar::make_exception_future<>(eptr);
+  return app.run(argc, argv, [&app] {
+      auto&& config = app.configuration();
```

Comment:
i'd suggest avoid the massive reindent unless we are changing the indented code at the same time.

Reply:
Sorry, didn't realize that this part doesn't change much. `test_messenger.cc` was originally rewritten to verify the sharded-messenger prototype, then supplemented with the original test cases.

Reply:
fixed.